docs/DeveloperGuide.md

```diff
@@ -273,6 +273,83 @@ the gender is not of valid format; Invalid person exception is thrown if the per
 contact list. Error message is displayed on the GUI subsequently.
 </div>
 
+### Add Date Of Birth
+
+The Add Date Of Birth feature allows users to add a date of birth field (format: dd/mm/yyyy) to a person in the contact list. It is performed as a part of `addPersonCommand#execute()`.
```

Comment:
contact list is a term we did not define in our DG and its we do have a class or field for it. Perhaps, we could use the term 'application' or 'addressbook' in place of the contact list and it'll be less confusing for first time readers of our DG/ users of our application?  

Also, if you're changing it from contact list, there are a few more instances of the phrase below which you will have to change too!